### PR TITLE
LPS-88439 Fix regression. We need to prevent a login popup via a SSO …

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
…service now that we are no longer hooking portal properties

@martamedio @arthurchan35  @csierra

### CODE REVIEW

File: modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/runtime/internal/SSOImpl.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/runtime/internal/SSOImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by stian-sigvartsen** [comment]
ci:test:saml

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:test:saml

**Comment by yichenroy** [comment]
Expediting the saml test suite, will be re-triggered on CI (github message will be posted soon).

**Comment by arthurchan35** [comment]
ci:test:saml

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**